dev-docs/bidders/kiviads.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: bidder
+title: Kivi
+description: Prebid Kivi Bidder Adapter
+biddercode: kiviads
+media_types: banner, video, native
+pbjs: false
+pbs: true
+safeframes_ok: true
+floors_supported: true
+fpd_supported: false
+multiformat_supported: will-not-bid
+ortb_blocking_supported: partial
```

Comment:
while we're waiting on the code review, I'd like to request that you add the rest of the missing metadata values as documented at https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#user-documentation -- in your case, the missing values are:
```
pbs_app_supported
gdpr_supported
usp_supported
coppa_supported
deals_supported
schain_supported
dchain_supported
```